Magento_Paypal/templates/express/review.phtml

```diff
@@ -0,0 +1,263 @@
+<?php
+/**
+ * @var \Magento\Paypal\Block\Express\Review $block
+ */
+?>
+<div class="paypal-review">
+    <div
+        class="
+            payapal-review__details
+            row
+            margin-bottom-m
+        "
+    >
+        <div class="block-content">
+            <?php if ($block->getShippingAddress()) : ?>
+                <div
+                    class="
+                        payapal-review__details-item
+                        col-lg-4
+                        margin-right-auto
+                        margin-bottom-m
+                    "
+                >
+                    <form
+                        method="post"
+                        id="shipping-method-form"
+                        action="<?= $block->escapeUrl($block->getShippingMethodSubmitUrl()) ?>"
+                        class="form"
+                    >
+                        <fieldset class="fieldset">
+                            <legend class="fieldset__legend fieldset__legend--hidden">
+                                <?= $block->escapeHtml(__('Shipping Method')) ?>
+                            </legend>
+                            <?php if ($block->canEditShippingMethod()) : ?>
+                                <?php if ($groups = $block->getShippingRateGroups()) : ?>
+                                    <?php $currentRate = $block->getCurrentShippingRate(); ?>
+                                    <label
+                                        class="label"
+                                        for="shipping-method"
+                                    >
+                                        <?= $block->escapeHtml(__('Shipping Method')) ?>
+                                    </label>
+                                    <div class="control">
+                                        <div
+                                            class="
+                                                select
+                                                select--native
+                                                field
+                                                shipping
+                                                required
+                                            "
+                                            required
```

Comment:
🚜

Reply:
I added `required` attribute for accessibility. Screen readers will announce that the field is required.

Reply:
please verify can we change this and is `required` readable by screen readers